.circleci/config.yml

```diff
@@ -42,15 +42,60 @@ jobs:
       - run:
           name: Build plugin artifact
           command: ./gradlew buildPlugin
+      - store_artifacts:
+          path: build/distributions/
+          destination: .
       - run:
           name: Verify plugin artifact contents
           command: ./scripts/verify-artifact-contents
       - run:
           name: Verify binary compatibility with supported IntelliJ versions
           command: ./scripts/verify-binary-compatibility
-      - store_artifacts:
-          path: build/distributions/
-          destination: .
+
+      - run:
+          # TODO (#274): use artifacts published on Bintray/JCenter instead
+          # Note: this step (at least building&publishing the ide-probe artifacts) can't be baked into the Docker image
+          # since the VirtuslabRnD/ide-probe is a private repository as for now,
+          # and also we can't publish the binary artifacts due to VL's client's policies.
+          name: Prepare prerequisites for UI tests
+          command: |
+            set -x
+            apt-get update
+            apt-get install -y gnupg
+            echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
+            curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add
+            apt-get update
+            apt-get install -y sbt
```

Comment:
Can't things up to this line be baked into Docker image?
It looks like regular software installation from repo.

Reply:
Yeah, could be... but I (naively) hope we soon won't need them anymore (i.e. we won't need to build ide-probe within the CI pipeline at all, but we'll use externally published artifacts instead, and hence sbt won't be needed at all)